### DISCUSSION TITLE
[@types/pdfmake] Single margins

### DISCUSSION THREAD

**Discussion by adros** [DefinitelyTyped/DefinitelyTyped#62329] [Issues with a @types package]
pdfmake support also using **single margins** (*marginTop* / *marginLeft* / ...)
It is not documented, but can be found here in code:

https://github.com/bpampuch/pdfmake/blob/0.2/src/docMeasure.js#L88

(if margin property is also used in any of the styles or directly on the node, theses single margins will be overriden)

So, I believe that the *interface Style* should be extended with these props.

[a comment by an automated account is not shown]

**Comment by kryops** [comment]
Seeing that https://github.com/bpampuch/pdfmake/issues/2482 received the "Documentation" tag, I think we can assume this is an undocumented public feature, and add it to the types.

I opened https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63134 for it.
